tools/congestion-model/src/workload/balanced.rs

```diff
@@ -96,7 +96,10 @@ impl BalancedProducer {
         assert!(!receipt_sizes.is_empty(), "must have at least one receipt size");
         assert!(execution_gas <= attached_gas, "must attach more gas than is needed for execution");
         let depth = receipt_sizes.len();
-        let num_receipts = fan_out.pow(depth as u32 - 1);
+        let mut num_receipts = 1;
+        for d in 1..depth {
+            num_receipts += fan_out.pow(d as u32);
+        }
```

Comment:
I noticed that I undercounted the total number of receipts and thus we would burn more gas as the sum of all receipts than what the parameters say.